### PR TITLE
Ado 12477 - Components A-C

### PR DESCRIPTION
**Pull request by sineade**
https://dev.azure.com/AMA-Ent/AMA-Ent/_workitems/edit/12477
This PR restyles all of the components from A-C

Accordions 
Screen Shot 2020-07-20 at 11 32 08 AM

Banners
Screen Shot 2020-07-20 at 11 33 11 AM

Breadcrumbs
Screen Shot 2020-07-20 at 10 26 27 AM

Buttons
Screen Shot 2020-07-20 at 1 45 03 PM

Button Groupings
Screen Shot 2020-07-20 at 1 44 34 PM

Screen Shot 2020-07-20 at 1 44 41 PM

Screen Shot 2020-07-21 at 2 55 28 PM

Screen Shot 2020-07-20 at 1 44 53 PM

Call to Action
Screen Shot 2020-07-20 at 10 27 24 AM

Callouts
Screen Shot 2020-07-20 at 11 35 05 AM

Cards
Screen Shot 2020-07-21 at 1 12 36 PM

Content Toggles
Screen Shot 2020-07-20 at 11 39 20 AM

Credit Card
Screen Shot 2020-07-20 at 10 28 06 AM

### CODE REVIEW

**Comment by amadeploy** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: Please provide a VSTS, ADO link or OPSGENIE link in the Pull Request description 💩. Updating the Danger gem might fix the issue. Your Danger version: 7.0.1, latest Danger version: 8.0.3

```
 #  from Dangerfile:11
 #  -------------------------------------------
 #    unless urls.any? { |url| github.pr_body.include?(url) }
 >      raise 'Please provide a VSTS, ADO link or OPSGENIE link in the Pull Request description 💩'
 #    end
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: app/assets/stylesheets/v3/layout_components/banners.scss [hunk 1]
```diff
@@ -1,27 +1,29 @@
 .banner {
   position: relative; // This is needed for the positioning of the close icon on the blue banners
   margin-bottom: $base-margin;
+  border-radius: $base-radius;
+  box-shadow: $base-box-shadow;
+  padding: $base-padding*2 $base-padding*3 $base-padding*2 $base-padding*3 !important;
```
Comment (kaytcampbell) [inline app/assets/stylesheets/v3/layout_components/banners.scss:6]: can we shorten this to padding: $base-padding\*2 $base-padding\*3?

File: app/assets/stylesheets/v3/layout_components/buttons/button-grouping.scss [hunk 2]
```diff
@@ -137,8 +150,17 @@
   }
 
   &__text {
-    line-height: 1rem;
+    line-height: 1.5;
```
Comment (kaytcampbell) [inline app/assets/stylesheets/v3/layout_components/buttons/button-grouping.scss:153]: this should have a unit?

**Comment by kaytcampbell** [comment]
@sineade one other question, for the button groupings, should we add the border to the bottom of them, like how we had mocked it up? 
image

File: app/assets/stylesheets/v2/_settings.scss [hunk 3]
```diff
@@ -207,6 +207,7 @@ $accordion-item-background-hover: $brand-blue-light;
 $accordion-item-padding: 1.25rem 1rem;
 $accordion-content-background: $white;
 $accordion-content-border: 0;
+$accordion-content-border-radius: 9px;
```
[the remark this answers is not in the document]
Comment (sineade) [reply app/assets/stylesheets/v2/_settings.scss:210]: nope. thank you.

File: app/assets/stylesheets/v2/layout_components/accordions.scss [hunk 4]
```diff
@@ -63,7 +63,6 @@
     padding-left: $base-padding;
 
     &--arrow {
-      font-size: $base-font-size*1.5;
```
[the remark this answers is not in the document]
Comment (sineade) [reply app/assets/stylesheets/v2/layout_components/accordions.scss:66]: noooppeee.

File: app/assets/stylesheets/v3/layout_components/buttons/button-grouping.scss (hunk 2 above)
Comment (sineade) [reply app/assets/stylesheets/v3/layout_components/buttons/button-grouping.scss:151]: Does it need it?

This is what I found about this subject:

"Unitless line heights are recommended due to the fact that child elements will inherit the raw number value, rather than the computed value. With this, child elements can compute their line heights based on their computed font size, rather than inheriting an arbitrary value from a parent that is more likely to need overriding."

File: app/assets/stylesheets/v3/layout_components/buttons/button-grouping.scss [hunk 5]
```diff
@@ -128,7 +141,6 @@
   &__bg-icon {
     height: 80px;
     background-repeat: no-repeat;
-    background-position: center center;
   }
 
   &__title {
```
[the remark this answers is not in the document]
Comment (sineade) [reply app/assets/stylesheets/v3/layout_components/buttons/button-grouping.scss:145]: updated screenshot

**Comment by sineade** [comment]
> @sineade one other question, for the button groupings, should we add the border to the bottom of them, like how we had mocked it up?
> image

I don't think we're going to be able to for phase 1, there isn't anything unique enough wrapping them to put the border on.
